examples/uros_6lowpan/uros_6lowpan_main.c

```diff
@@ -0,0 +1,197 @@
+#include <nuttx/config.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+//uROS libraries
+#include <rcl/rcl.h>
+#include <rcl/error_handling.h>
+#include <std_msgs/msg/int32.h>
+#include <rmw_uros/options.h>
+
+#if defined(BUILD_MODULE)
+int main(int argc, char *argv[])
+#else
+int uros_6lowpan_main(int argc, char* argv[])
+#endif
+{
+    char buffer[256]; // Buffer to save configuration commands.
+    char aux_buffer[2]; // Buffer to save the configuration ID.
+
+    if(3 > argc || 0 == atoi(argv[2]))
+    {
+        printf("usage: program [-h | --help] | ip port sub/pub [<max_topics>]\n");
+        return 0;
+    }
+    //Check if the IP and the port are correct
+    if(strlen(argv[1])>39 || strlen(argv[2])>4){
+        printf("Error: IP or port size incorrect \r\n");
+        return 0;
+    }
+
+    //6lowpan configuration process
+    printf("Do you want to configure the 6lowpan network? (Y/N)\r\n");
+    memset(buffer,0,sizeof(buffer));
+    scanf("%s", buffer);
```

Comment:
Include length specifier:
```suggestion
    scanf("%255s", buffer);
```